README.md

```diff
@@ -1,5 +1,7 @@
 # Payone Module
 
+[![Build Status](https://travis-ci.org/spryker-eco/payone.svg?branch=feature/ECO-508-ci-setup)](https://travis-ci.org/spryker-eco/payone)
```

Comment:
I would set the master here always and never change it.

Reply:
done!